homeassistant/components/sense/binary_sensor.py

```diff
@@ -73,10 +96,12 @@ def sense_to_mdi(sense_icon):
 class SenseDevice(BinarySensorDevice):
     """Implementation of a Sense energy device binary sensor."""
 
-    def __init__(self, data, device):
+    def __init__(self, data, device, sense_monitor_id):
         """Initialize the Sense binary sensor."""
         self._name = device["name"]
         self._id = device["id"]
+        self._sense_monitor_id = sense_monitor_id
+        self._unique_id = f"sense-{sense_monitor_id}-{self._id}"
```

Comment:
Unique IDs are scoped to integration (binary_sensor) + platform (sense). So no need to include `sense-`:
```suggestion
        self._unique_id = f"{sense_monitor_id}-{self._id}"
```

Reply:
excellent. removing sense-